.github/workflows/integration.yml

```diff
@@ -0,0 +1,61 @@
+name: Integration Tests
+on:
+  push
+jobs:
+  run-integration-tests:
+    runs-on: ubuntu-latest
+    environment: azure-prod-pecou
+    strategy:
+      matrix:
+        # python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
```

Comment:
Waiting to enable for the whole matrix until I can see it working for 3.7 and 3.8